third-party/cmake/BuildGperf.cmake

```diff
@@ -2,6 +2,7 @@
 # cross compiling we still want to build for the HOST system, whenever
 # writing a recipe that is meant for cross-compile, use the HOSTDEPS_* variables
 # instead of DEPS_* - check the main CMakeLists.txt for a list.
+project(NVIM_DEPS_GPERF C CXX)
```

Comment:
is each `cmake/foo.cmake` already formally a "sub project"? Or is this a new subproject?

Reply:
According to https://cmake.org/cmake/help/latest/command/project.html it might mess with `PROJECT_NAME` (I will check it after the `include`).

Reply:
https://cmake.org/cmake/help/latest/command/enable_language.html?highlight=language#command:enable_language seems better, fixed.